tests/testdata/modules/site-health/audit-autoloaded-options/class-autoloaded-options-mock-responses.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Class Autoloaded_Options_Mock_Responses mock site status health data for autoloaded options.
+ *
+ * @since 1.0.0
+ */
+class Autoloaded_Options_Mock_Responses {
```

Comment:
I see the reasoning to have this is in a separate class, however it seems like the only reason to exists is to return an array in all cases, it seems like a `@dataprovider` would be enough for all the method using this class:

- https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html

That would remove the need to have this class in the first place.

Reply:
@mitogh from the 3 methods of that class, 2 accept arguments. Though this can be achievable with providers, I think for simplicity and readability's sake, I prefer the class choice.

Reply:
Thank you @manuelRod let me take a look